src/DrinkDropdown.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+export default function DrinkDropdown({ setDrinkId }) {
+  function handleChange(e) {
+    setDrinkId(e.target.value);
+  }
+
+  return <select onChange={handleChange}>
```

Comment:
you can call your state setter directly in your onChange without defining this additional function on line 4
`onChange={(e) => setDrinkId(e.target.value)}`